mscore/editstyle.cpp

```diff
@@ -420,6 +424,27 @@ EditStyle::EditStyle(Score* s, QWidget* parent)
       comboFBFont->setCurrentIndex(0);
       connect(comboFBFont, SIGNAL(currentIndexChanged(int)), SLOT(on_comboFBFont_currentIndexChanged(int)));
 
+      // chord symbol init
+      harmonyPlay->setChecked(true);
+
+      voicingSelectWidget->interpretBox->clear();
+      voicingSelectWidget->interpretBox->addItem(tr("Jazz"), int(0)); // two-item combobox for boolean style variant
+      voicingSelectWidget->interpretBox->addItem(tr("Literal"), int(1)); // true = literal
+
+      voicingSelectWidget->voicingBox->clear();
+      voicingSelectWidget->voicingBox->addItem(tr("Automatic"), int(Voicing::AUTO));
+      voicingSelectWidget->voicingBox->addItem(tr("Root Only"), int(Voicing::ROOT_ONLY));
+      voicingSelectWidget->voicingBox->addItem(tr("Close"), int(Voicing::CLOSE));
+      voicingSelectWidget->voicingBox->addItem(tr("Drop 2"), int(Voicing::DROP_2));
+      voicingSelectWidget->voicingBox->addItem(tr("Six Note"), int(Voicing::SIX_NOTE));
+      voicingSelectWidget->voicingBox->addItem(tr("Four Note"), int(Voicing::FOUR_NOTE));
+      voicingSelectWidget->voicingBox->addItem(tr("Three Note"), int(Voicing::THREE_NOTE));
```

Comment:
Maybe instead of casting to `int` it'd be better to use `QVariant::fromValue<Voicing>(Voicing::THREE_NOTE)` etc.?

Reply:
Maybe, can you help me understand why?

Reply:
I've seen it done elsewhere, and that method asked for a `QVariant` as the 2nd arg. If does know `int` and deals with that, but this then is a double cast, first explicitly to `int`, then implicitly to `QVariant`.But I see that it is done your way a lot in that file.

Reply:
I think it's not necesssary. `int` is more readable

Reply:
readable and correct are different things though, `QComboBox::addItem(const QString &text, const QVariant &userData = QVariant())` is the method's signature

Reply:
Strangely doing that results in a rather bizarre compiler error

Reply:
C:\Qt\5.12.8\mingw73_64\include\QtCore\qglobal.h:121: Fehler: static assertion failed: Type is not registered, please use the Q_DECLARE_METATYPE macro to make it known to Qt's meta-object system
 #  define Q_STATIC_ASSERT_X(Condition, Message) static_assert(bool(Condition), Message)
                                                 ^

Reply:
And it wouldn't work in this case... so indeed `int() is fine`